modules/react-components/src/components/header/header.tsx

```diff
@@ -215,14 +215,20 @@ export const Header: FunctionComponent<HeaderPropsInterface> = (
     };
 
     const resolveAuthenticatedUserEmail = (): string => {
-        if (!profileInfo || !profileInfo.emails || !profileInfo.emails.length || profileInfo.emails.length < 1) {
+        if (!profileInfo || !profileInfo.userName) {
             return null;
         }
 
-        if (typeof profileInfo.emails[ 0 ] === "string") {
-            return profileInfo.emails[ 0 ];
-        } else if (typeof profileInfo.emails[ 0 ] === "object") {
-            return profileInfo.emails[ 0 ].value;
+        if (typeof profileInfo.userName === "string") {
+            const userName = profileInfo?.userName?.split("/").length > 1
+            ? profileInfo?.userName?.split("/")[1]
```

Comment:
Since we have checked for null in the condition, we can omit it here.
```suggestion
            ? profileInfo.userName.split("/")[1]
```